tests/e2e/core-tests/specs/merchant/wp-admin-order-new.test.js

```diff
@@ -4,12 +4,196 @@
  */
 const {
 	merchant,
-	verifyPublishAndTrash
-} = require( '@woocommerce/e2e-utils' );
+	verifyPublishAndTrash,
+	uiUnblocked
+} = require('@woocommerce/e2e-utils');
+const config = require('config');
+const {
+	HTTPClientFactory,
+	VariableProduct,
+	GroupedProduct,
+	SimpleProduct,
+	ProductVariation,
+	ExternalProduct
+} = require('@woocommerce/api');
+
+const taxRates = [
+	{
+		name: 'Tax Rate Simple',
+		rate: '10',
+		class: 'tax-class-simple'
+	},
+	{
+		name: 'Tax Rate Variable',
+		rate: '20',
+		class: 'tax-class-variable'
+	},
+	{
+		name: 'Tax Rate External',
+		rate: '30',
+		class: 'tax-class-external'
+	}
+];
+
+const taxTotals = ['£10.00', '£40.00', '£240.00'];
```

Comment:
@rodel-calasagsag The currency used in the e2e tests is USD. You can initialize your test environment to a consistent state with `npx wc-e2e test:e2e tests/e2e/specs/activate-and-setup/setup-onboarding.test.js` after doing `docker:up`.

Reply:
Got it. Thanks @rrennick. I'll do that.

Reply:
Done, I already changed the expected currency to USD.